tests/functional/spec/save_sign_out.spec.js

```diff
@@ -49,6 +49,25 @@ describe("Save sign out / Exit", () => {
     await expect(await browser.getUrl()).to.contain("thank-you");
   });
 
+  it("Given a I have started a social questionnaire, when I select save and sign out, then I am redirected to the signed out page and the correct access code link is shown", async () => {
+    await browser.openQuestionnaire("test_theme_social.json", { version: "v2", theme: "social" });
+    await expect(await $(SubmitPage.saveSignOut()).getAttribute("href")).to.contain("/sign-out?internal_redirect=True");
+    await $(SubmitPage.saveSignOut()).click();
+    await expect(await browser.getUrl()).to.contain("/signed-out");
+    await expect(await $("body").getHTML()).to.contain("Your progress has been saved");
+    await expect(await $("body").getHTML()).to.contain("To resume the survey,");
+  });
+
+  it("Given a I have started a business questionnaire, when I select save and sign out, then I am redirected to the signed out page and the correct access code link is shown", async () => {
```

Comment:
I think the sign in link should be asserted for the different survey types - unless its subject to change soon for the business ones to change form todo to something else

Reply:
The todo link is tested in a test above and both survey types have now been covered with the 'internal-redirect` asserts